### PR TITLE
Update based on latest spec

### PR DESCRIPTION
**Pull request by jlid87**

### CODE REVIEW

File: source/documentation/get-all-messages.html.md.erb [hunk 1]
```diff
@@ -11,13 +11,19 @@ Use the GET endpoint [Get all messages for an excise movement using the movement
 
 The consignor ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
 
+## Path parameters
+
+Parameters must be entered in lowercase.
+
+| Parameter | Type | Details |
+| -- | -- | -- |
+| lrn | String | Filter messages by local reference number (LRN) |
```
Comment (mpbostock) [inline source/documentation/get-all-messages.html.md.erb:20]: The details of this should probably be something like "The local reference number of the movement"

File: source/documentation/submit-message.html.md.erb [hunk 2]
```diff
@@ -15,6 +15,14 @@ The X-Message-Type header **must** be the same as in the message body.
 
 The consignor ID or consignee ID must match an excise reference number (ERN) in the bearer token that you use for API authentication. If it does not, a 403 Forbidden error will be thrown.
 
+## Path parameters
+
+Parameters must be entered in lowercase.
+
+| Parameter | Type | Details |
+| -- | -- | -- |
+| lrn | String | Filter messages by local reference number (LRN) |
```
Comment (mpbostock) [inline source/documentation/submit-message.html.md.erb:24]: The details of this should probably be something like "The local reference number of the movement"

**Review by mpbostock — CHANGES_REQUESTED** [review]

File: source/documentation/get-all-messages.html.md.erb [hunk 3]
```diff
@@ -11,13 +11,19 @@ Use the GET endpoint [Get all messages for an excise movement using the movement
...
+
 ## Query parameters
 
 | Parameter | Type | Details |
 | -- | -- | -- |
-| Date range | String | Filter messages by date range |
-| Message type| String | Filter messages by message type |
-| LRN | String | Filter messages by local reference number (LRN)|
+| updatedSince | String <date-time> (DateTime) | Filter excise movements modified after provided timestamp. This can be used to poll for new messages per ERN/LRN combination.<br></br><br></br> You can also pollon the [Get excise movements endpoint](get-excise-movements.html) which allows a more efficient way to poll as that can first give you a list of movements that have new messages, and then use this endpoint to retrieve the updated messages for those movements.|
```
Comment (ChrisCookOC) [inline source/documentation/get-all-messages.html.md.erb:26]: "pollon"

File: source/documentation/get-all-messages.html.md.erb (hunk 1 above)
Comment (jlid87) [reply source/documentation/get-all-messages.html.md.erb:20]: Thanks

File: source/documentation/submit-message.html.md.erb (hunk 2 above)
Comment (jlid87) [reply source/documentation/submit-message.html.md.erb:24]: Thanks

File: source/documentation/get-all-messages.html.md.erb (hunk 3 above)
Comment (jlid87) [reply source/documentation/get-all-messages.html.md.erb:26]: Thanks

**Review by mpbostock — APPROVED** [review]

**Pull request merged**